core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntityProvider.scala

```diff
@@ -240,9 +240,13 @@ class CompactEntityProvider(requestArguments: EntityRequestArguments,
   }
 
   override def listEntities(entityType: String): Source[Entity, NotUsed] =
-    throw new DataEntityException("list all entities not supported for compact data tables.",
-                                  code = StatusCodes.NotImplemented
-    )
+    Source
+      .future(repository.dataSource.inTransaction(ReadOnly) { _ =>
```

Comment:
As written, this loads everything from the db into memory before returning it to the caller. But I think the desire (given that the return type is `Source`) is to stream it. [Something like this](https://github.com/broadinstitute/rawls/blob/43320ce495ea442f6bae4ff32a672ee6a3d4d644/core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/entityQuery/EntityQueryStrategy.scala#L33).